test/chrome_test.js

```diff
@@ -101,13 +101,37 @@ describe("StorageArea", function() {
       }); // give time for call to stubbed sync method to complete
     });
 
-    it("should respect MIN_INTERVAL of 1000ms", function(done) {
-      area.config = { type: "kinto", interval: 123 };
-      expect(consoleSpy
-          .calledWith("Sync interval should be at least 1000 milliseconds"))
-          .to.eql(true);
-      expect(syncStub.called).to.eql(false);
-      done();
+    it("checks config is an object", function(done) {
+      try {
+        area.config = 42;
+      } catch(err) {
+        expect(err.message).to.eql(
+            "config should be an object");
+        expect(syncStub.called).to.eql(false);
+        done();
+      }
+    });
+
+    it("checks type equals \"kinto\"", function(done) {
+      try {
+        area.config = { type: "foo", interval: 123000 };
+      } catch(err) {
+        expect(err.message).to.eql(
+            "Unsupported type, try setting chrome.storage.sync.config.type = \"kinto\".");
+        expect(syncStub.called).to.eql(false);
+        done();
+      }
+    });
+
+    it("checks MIN_INTERVAL of 1000ms", function(done) {
+      try {
+        area.config = { type: "kinto", interval: 123 };
+      } catch(err) {
+        expect(err.message).to.eql(
+            "Sync interval should be at least 1000 milliseconds");
+        expect(syncStub.called).to.eql(false);
+        done();
+      }
```

Comment:
You can test for thrown errors that way:

```js
expect(function(){
  area.config = { type: "kinto", interval: 123 };
}).to.Throw(Error, /at least 1000/);
```

Also no need for async test here I think.